### PR TITLE
Chronos: unset LD_PRELOAD automatically for autots

### PR DESCRIPTION
**Pull request by Rebekk**

### CODE REVIEW

**Comment by jason-dai** [comment]
We should submit an issue and track it at https://github.com/intel-analytics/BigDL/issues/4377 @TheaperDeng

File: python/chronos/test/bigdl/chronos/autots/__init__.py [hunk 1]
```diff
@@ -13,3 +13,19 @@
...
+
+try:
+    from .autotsestimator import AutoTSEstimator
+except ImportError:
+    warnings.warn("Please install `bigdl-nano[all]` to use AutoTSEstimator")
+
+try:
+    LD = os.environ["LD_PRELOAD"]
+except:
+    warnings.warn("No need to unset variables")
```
Comment (TheaperDeng) [inline python/chronos/test/bigdl/chronos/autots/__init__.py:28]: we don't need to explicitly warning our users about this

File: python/chronos/test/bigdl/chronos/autots/test_init.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import pytest
+from unittest import TestCase
+import os
+
+class TestLD(TestCase):
+    def test_ld_preload(self):
+        try:
+            LD = os.environ["LD_PRELOAD"]
+        except:
+            pass
```
Comment (TheaperDeng) [inline python/chronos/test/bigdl/chronos/autots/test_init.py:27]: we may need to
1. set LD_PRELOAD
2. import autots
3. check if it has been successfully unset

**Review by TheaperDeng — APPROVED** [review]
currently LGTM

**Comment by TheaperDeng** [comment]
just had a thought, LD_PRELOAD might be useful when it comes to `TSPipeline`.

**Comment by TheaperDeng** [comment]
well, just run jenkins for this pr and we can merge it first

**Comment by Rebekk** [comment]
jenkins：
http://10.112.231.51:18888/job/BigDL-Chronos-PR-Validation/1073/
http://10.112.231.51:18888/job/BigDL-PRVN-chronos-Python-Spark-3.1-ray-tf1/1113/
http://10.112.231.51:18888/job/BigDL-PRVN-chronos-Python-Spark-3.1-py37-ray-part2/1085/

**Pull request closed without merging**